nixos/tests/systemd-networkd-ipv6-prefix-delegation.nix

```diff
@@ -40,7 +40,8 @@ import ./make-test-python.nix ({ pkgs, lib, ... }: {
             address = [
               "2001:DB8::1/64"
             ];
-            networkConfig.IPForward = true;
+            networkConfig.IPv4Forwarding = true;
```

Comment:
I wonder if this setting (`IPv4Forwarding`) is actually needed for this IPv6 test.